lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1793,6 +1793,29 @@ definitions:
                 example:
                   1: ['192.168.0.0/16']
                   2: ['172.10.0.0/16', '172.20.0.0/16']
+      restricted_plmn_ids:
+        type: array
+        description: 'List of PLMN IDs restricted in the network'
+        items:
+          type: string
+          minLength: 5
+          maxLength: 6
+          pattern: '^(\d{5,6})$'
+          example: '123456'
+      service_area_maps:
+        type: object
+        description: 'Mapping service areas to tacs in the network'
+        additionalProperties:
+          $ref: '#/definitions/tac_list'
```

Comment:
This is not a map though. It should be  an array of service_area_id: [tac1, tac2], e.g.,

             example:
               120:
               - 1
               - 2
               240:
               - 2
               - 10
               - 32